### PR TITLE
Zarf walkthrough update

### PR DESCRIPTION
**Pull request by laurenmengert**
## Description

- Updated the 'Using zarf package create' walkthrough
- Changed both the 'Using zarf package create' and 'Deploying zarf packages' walkthroughs to use the helm-oci-chart example
- Abstracted the dos games example from the 'create' and 'deploy' walkthroughs to create a full start to finish walkthrough
- Fixed some grammar
- Standardized some styling 
- Changed order of some pages (I wanted the full dos games walkthrough to come right after the 'deploying zarf packages' one. That seemed to make the most sense to me)

*See comments section for some questions/requested feedback*
...

## Related Issue

Fixes #

Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

**Comment by Racer159** [comment]
> * Regarding the full Dos Games walkthrough - do we think I should add running zarf init? Do we want to make that a true walkthrough of every step, or is just the creation and deployment of the dos-games package enough?
> 
>     * I'd like opinions on whether or not we should keep the video walkthrough that is in the Dos-games section. That video only covers the deployment of the package, whereas the walkthrough article covers the creation _and_ deployment
> 
>     * There was something else but I can't remember.. It will come to me and I'll make another comment or edit this one

I think linking init as a pre-req is fine and we can just focus on create/deploy.

IMO the dos-games video should likely be updated to include both create and deploy and more closely match the walkthrough.

(also there are merge conflicts on this branch that need to be resolved)

File: docs/13-walkthroughs/2-deploying-zarf-packages.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#### Remediation
+
+If you receive this error when zarf is attempting to deploy the `BASELINE COMPONENT`, this means you have not initialized the kubernetes cluster.  This is one of the prerequisites for this walkthrough.  Perform the [Initialize a cluster](./1-initializing-a-k8s-cluster.md) walkthrough, then try again.
+
+## Credits
+
+:sparkles: Special thanks to these fine references! :sparkles:
+
+- <https://www.reddit.com/r/programming/comments/nap4pt/dos_gaming_in_docker/>
+- <https://earthly.dev/blog/dos-gaming-in-docker/>
```
Comment (Racer159) [inline docs/13-walkthroughs/2-deploying-zarf-packages.md:101]: This isn't needed on the helm one since it is only for the dos walkthrough

**Review by Racer159 — CHANGES_REQUESTED** [review]
Just a few small suggestions.

**Comment by Racer159** [comment]
Also looks like there are a few broken links as well
image

File: docs/13-walkthroughs/2-deploying-zarf-packages.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+### Secrets "zarf-state" not found
+
+#### Example
+
+<iframe src="/docs/walkthroughs/troubleshoot_uninitialized.html" width="100%" height="250px"></iframe>
+
+#### Remediation
+
+If you receive this error when zarf is attempting to deploy the `BASELINE COMPONENT`, this means you have not initialized the kubernetes cluster.  This is one of the prerequisites for this walkthrough.  Perform the [Initialize a cluster](./1-initializing-a-k8s-cluster.md) walkthrough, then try again.
```
[the remark this answers is not in the document]
Comment (laurenmengert) [reply docs/13-walkthroughs/2-deploying-zarf-packages.md:94]: Whoops, good catch

**Comment by Madeline-UX** [comment]
> > * Regarding the full Dos Games walkthrough - do we think I should add running zarf init? Do we want to make that a true walkthrough of every step, or is just the creation and deployment of the dos-games package enough?
> >   
> >   * I'd like opinions on whether or not we should keep the video walkthrough that is in the Dos-games section. That video only covers the deployment of the package, whereas the walkthrough article covers the creation _and_ deployment
> >   * There was something else but I can't remember.. It will come to me and I'll make another comment or edit this one
> 
> I think linking init as a pre-req is fine and we can just focus on create/deploy.
> 
> IMO the dos-games video should likely be updated to include both create and deploy and more closely match the walkthrough.
> 
> (also there are merge conflicts on this branch that need to be resolved)

I have thought about this some - I think if we were to provide the game as pre-built (on release page) it could all be in one. I struggle with the walkthrough being build and deploy because for a lot of users those tasks will be separate. Although I know package creators will deploy to test. I think the bigger issue is the dos games is an example package. So this should really be how to build a package and how to deploy a package (general steps for packages)

**Comment by Racer159** [comment]
> > > * Regarding the full Dos Games walkthrough - do we think I should add running zarf init? Do we want to make that a true walkthrough of every step, or is just the creation and deployment of the dos-games package enough?
> > >   
> > >   * I'd like opinions on whether or not we should keep the video walkthrough that is in the Dos-games section. That video only covers the deployment of the package, whereas the walkthrough article covers the creation _and_ deployment
> > >   * There was something else but I can't remember.. It will come to me and I'll make another comment or edit this one
> > 
> > 
> > I think linking init as a pre-req is fine and we can just focus on create/deploy.
> > IMO the dos-games video should likely be updated to include both create and deploy and more closely match the walkthrough.
> > (also there are merge conflicts on this branch that need to be resolved)
> 
> I have thought about this some - I think if we were to provide the game as pre-built (on release page) it could all be in one. I struggle with the walkthrough being build and deploy because for a lot of users those tasks will be separate. Although I know package creators will deploy to test. I think the bigger issue is the dos games is an example package. So this should really be how to build a package and how to deploy a package (general steps for packages)

I agree though I think I'd like to see https://github.com/defenseunicorns/zarf/issues/758 done first to really build that out - we have the dos games available via sget but this is not likely to be something that most people use as the OCI functionality is much easier for most to use.

**Review by Racer159 — APPROVED** [review]

**Pull request merged**